lib/oli/lti/platform_external_tools.ex

```diff
@@ -178,11 +180,21 @@ defmodule Oli.Lti.PlatformExternalTools do
         dynamic([p, lad], lad.status == :enabled)
       end
 
-    # TODO: calculate usage_count (we need https://eliterate.atlassian.net/browse/MER-4469)
+    usage_counts_query =
```

Comment:
I might be missing something, but here we are counting the number of activity registrations associated with a platform instance. Does this number represent the usage count?

Reply:
In testing this out, I noticed the same issue. The number being displayed does not represent the actual usage counts displayed in the table it links to

Reply:
Thanks for pointing that out. Fixed in b4ee9b81540f4e48307fcb0db81b0ce0ee45da1c